### PR TITLE
fix tags for 'intentional' errors : (python_essentials.rst)

### PR DESCRIPTION
**Pull request by fkazemian**
- [x] ./rst_files/python_essentials.rst: :class: no-execute  #98
- [x] ./rst_files/python_essentials.rst: :class: no-execute
screenshot from 2018-10-26 11-16-09
```
In this case you can shift the file to the pwd or specify the `full path <https://en.wikipedia.org/wiki/Path_%28computing%29>`_ to the file

.. code-block:: python3
    :class: no-execute

    f = open('insert_full_path_to_file/newfile.txt', 'r')
```

- [x] ./rst_files/python_essentials.rst: :class: no-execute 
screenshot from 2018-10-26 11-19-52
```
If you did the exercises in the :ref:`previous lecture <python_by_example>`, you would have come across the statement

.. code-block:: python3
    :class: no-execute

    plt.plot(x, 'b-', label="white noise")

In this call to Matplotlib's ``plot`` function, notice that the last
```
-----------------------------------------------------------------------------------------------------------

1:
```
    x = (1, 2)  
    x[0] = 10  

```
should be executed and raise an Error but shouldn't be counted in any coverage tests. So I change 
```
But tuples are not

.. code-block:: python3
    :class: no-execute

    x = (1, 2)  
    x[0] = 10  

.. code-block:: none

    ---------------------------------------------------------------------------
    TypeError                                 Traceback (most recent call last)
    <python-input-21-6cb4d74ca096> in <module>()
    ----> 1 x[0]=10

    TypeError: 'tuple' object does not support item assignment


We'll say more about the role of mutable and immutable data a bit later
```
to 
```
But tuples are not

.. code-block:: python3
    :class: skip-test

    x = (1, 2)  
    x[0] = 10  


We'll say more about the role of mutable and immutable data a bit later
```

screenshot from 2018-10-26 11-12-48
------------
2: add     :class: no-execute  to .. code-block:: none
screenshot from 2018-10-26 11-25-36
```

.. code-block:: none
    :class: no-execute

    Type:       function
    String Form:<function f at 0x2223320>
    File:       /home/john/temp/temp.py
    Definition: f(x)
    Docstring:  This function squares its argument

.. code-block:: ipython

    f??

.. code-block:: none
    :class: no-execute

    Type:       function
    String Form:<function f at 0x2223320>
    File:       /home/john/temp/temp.py
    Definition: f(x)
    Source:
    def f(x):
        """
        This function squares its argument
        """
        return x**2

```
------------------
Issue  #85

### CODE REVIEW

File: rst_files/python_essentials.rst [hunk 1]
```diff
@@ -698,7 +689,8 @@ After running this code, the docstring is available
 
     f?
 
-.. code-block:: none
+.. code-block:: ipython
+    :class: no-execute
```
[the remark this answers is not in the document]
Comment (natashawatkins) [reply rst_files/python_essentials.rst]: But it's not being executed right? the output is manually copied in?

**Comment by mmcky** [comment]
@natashawatkins would you mind to copy ``python_essentials.rst`` - close this PR and reopen a new PR with just python_essentials in it?

**Comment by natashawatkins** [comment]
will do

File: rst_files/python_essentials.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (mmcky) [reply rst_files/python_essentials.rst]: that's correct. But the executed notebook won't show the result as it is not contained in an `out` block

[the remark this answers is not in the document]
Comment (mmcky) [reply rst_files/python_essentials.rst]: this is a tricky edge case.

[the remark this answers is not in the document]
Comment (mmcky) [reply rst_files/python_essentials.rst]: it ends up looking like this: 
image

**Comment by natashawatkins** [comment]
open here #115

**Pull request closed without merging**